WPAndroid Integration.md

```diff
@@ -12,6 +12,8 @@ To do this, we need to copy the [local-builds.gradle-example file](https://githu
 Building the project as such will make WPAndroid build the source code for `react-native-bridge`, `react-native-aztec` and other sublibraries from the locally checked out folder.
 It'll also expect the metro server to be running to fetch the JS bundle.
 
+**Note:** The `ci/circleci: Build Android RN Bridge & Publish to Bintray` task in `gutenberg-mobile` needs to complete before WPAndroid's `Installable Build` CI task will succeed. As such, you may see WPAndroid's CI tasks fail with a 403 error if they run before the `gutenberg-mobile` task completes. Before deploying any changes to `ext.gutenbergMobileVersion`, either wait for the tests to pass in `gutenberg-mobile` first or restart the WPAndroid CI tasks after they pass.
```

Comment:
I think this note is likely best suited within the ["Update WPAndroid's binary version" section](https://github.com/wordpress-mobile/gutenberg-mobile/blob/c2fff355b068087a31df37ffe6042647e8a44a5d/WPAndroid%20Integration.md#update-wpandroids-binary-version), as it directly pertains the steps covered there. WDYT? 

Additionally, I made a few nit changes for clarity. Feel free to ignore them if you disagree. 

```suggestion
**Note:** The `Build Android RN Bridge & Publish to Bintray` task in `gutenberg-mobile` needs to complete before WPAndroid's `Installable Build` CI task will succeed. As such, you may see WPAndroid's CI tasks fail with a 403 error if they run before the `gutenberg-mobile` task completes. You can either wait for wait for `gutenberg-mobile`'s CI task to pass before pushing changes to `ext.gutenbergMobileVersion` or restart failed WPAndroid CI tasks after `gutenberg-mobile`'s CI task passes.
```

Reply:
Great, thank you for those clarifications!

> I think this note is likely best suited within the "Update WPAndroid's binary version" section, as it directly pertains the steps covered there. WDYT?

Agreed, I _almost_ went there as well, but got swayed by the fact some of the sections also deal with changes to `builds.gradle`. I think you're right that it's most relevant to this section, though, and will place it at the bottom.